### DISCUSSION TITLE
How do I create more pages using the app router?

### DISCUSSION THREAD

**Discussion by Elindo586** [vercel/next.js#68037] [Help] [answered]
### Summary

Silly question, but apparently I am getting error 
`404 | This page could not be found.`

https://github.com/Elindo586/estuff/tree/main/app
^^^  From link above, I am trying to access `page2.js`  as  `http://localhost:3000/page2.js `

Right now page2.js is shown at the root directory, but originally I tried to put it under app/file/page2.js , and still gave me the 404 Error when I tried to navigate to:  `http://localhost:3000/file/page2`

BTW.. my terminal keeps giving me a 
`GET /sw.js 404 in 41ms`   <---what is that??

**Comment by icyJoseph** [comment, marked as answer]
Change it to `page2/page.js`

In the app router, the folder defines the route name, and its contents are defined by certain files. 

- https://nextjs.org/docs/app/api-reference/file-conventions/page
- https://nextjs.org/docs/app/building-your-application/routing/pages-and-layouts

This is fundamentally different from Pages Router, where the file was the end of the route. So `pages/page2.js` would be `/page2` as a route, but in App Router `app/page2/page.js` would be the route.

And yes, you'll end up with a bunch of `page.js` files around. If you are using VSCode, I can recommend finding this editor setting: `labelFormat` and setting it to `short`.
